cmd/kapacitor/main.go

```diff
@@ -2246,3 +2258,48 @@ func doBackup(args []string) error {
 	}
 	return nil
 }
+
+func watchUsage() {
+	var u = `Usage: kapacitor watch <output file>
```

Comment:
I think writing to Stdout is fine, no need for an output file. Also allowing optional list of key value pairs after the task id would be nice. That way you could filter down by node or whatever within a task easily.

Reply:
typo :) fixed.